ludwig/schema/features/augmentation/utils.py

```diff
@@ -0,0 +1,157 @@
+from dataclasses import field
+
+from marshmallow import fields, ValidationError
+from marshmallow_dataclass import dataclass
+
+from ludwig.api_annotations import DeveloperAPI
+from ludwig.constants import TYPE
+from ludwig.schema import utils as schema_utils
+from ludwig.schema.features.augmentation.base import BaseAugmentationContainerConfig
+from ludwig.utils.registry import Registry
+
+
+@DeveloperAPI
+@dataclass(repr=False)
+class ImageAugmentationContainerConfig(BaseAugmentationContainerConfig):
+    """Augmentation container for image features."""
+
+    pass
+
+
+# TODO: Is all of this needed?
+# augmentation_registry = Registry()
+#
+#
+# @DeveloperAPI
+# def register_augmentation(name: str):
+#     def wrap(augmentation_config: BaseAugmentationConfig):
+#         augmentation_registry[name] = augmentation_config
+#         return augmentation_config
+#
+#     return wrap
+
+_augmentation_config_registry = Registry()
+
+
+@DeveloperAPI
+def get_augmentation_config_registry() -> Registry:
+    return _augmentation_config_registry
+
+
+def register_augmentation_config(name: str):
+    def wrap(cls):
+        get_augmentation_config_registry()[name] = cls
+        return cls
+
+    return wrap
+
+
+def get_augmentation_cls(name: str):
+    return get_augmentation_config_registry()[name]
+
+
+@DeveloperAPI
+def AugmentationContainerDataclassField(default=None, description=""):
```

Comment:
The `default` should be `[]` (or some kind of list).

Reply:
done